tests/test_panel.py

```diff
@@ -0,0 +1,78 @@
+import asyncio
+import pytest
+import aiohttp_jinja2
+import pathlib
+from aiohttp_debugtoolbar.panels.base import DebugPanel
+from aiohttp import ClientSession
+
+
+@pytest.mark.run_loop
```

Comment:
Master has changed pytest utility for aiohttp code.
Please replace decorator with `@asyncio.coroutine`